src/utils/toast.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const defualtToastSettings = {
+  position: "bottom-center",
+  autoClose: 3000,
+  hideProgressBar: true,
+  closeOnClick: true,
+  pauseOnHover: false,
+  draggable: false,
+  progress: undefined,
+  theme: "colored",
+};
+
+const toastStyles = {
+  padding: "2em",
+  marginBottom: "4em",
+};
+
+function ToastBox() {
+  return <ToastContainer style={toastStyles} />;
+}
+
+function ToastError(msg, options = {}) {
```

Comment:
Nazywanie funkcji z wielkiej litery może być trochę mylące i mieszać nam się z komponentami. Co powiesz na to, żeby nazwać tę funkcję z małej litery i dodać czasownik, żeby na pierwszy rzut oka dało się zrozumieć co się wydarzy, gdy zawołamy tę funkcję, np.
```suggestion
function showErrorToast(msg, options = {}) {
```
Co o tym myślisz?

Reply:
Jasne, dobry pomysł, tak też zrobiłem 😃